bentoml/xgboost.py

```diff
@@ -213,7 +212,7 @@ def num_concurrency_per_replica(self):
     @property
     def num_replica(self):
         if self.resource_quota.on_gpu:
-            return self.resource_quota.gpu
+            return self.resource_quota.gpus
```

Comment:
`len(self.resource_quota.gpus)`

Reply:
Since gpus is a list now right? Also we might need to look into tpus support as well